### DISCUSSION TITLE
Welcome to BeamMP Discussions!

### DISCUSSION THREAD

**Discussion by Starystars67** [Starystars67/BeamNG-MP#167] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by NCAsquad** [comment]
ok so there is the 0.24 update and there is no multiplayer button anymore. So could they fix that?

**Reply by Starystars67** [reply]
Hello, Clearing the cache should solve that. Also make sure that the mod is enabled in the repository>mods area.
